### DISCUSSION TITLE
Why Is It Not Possible to Specify RedirectUri for

### DISCUSSION THREAD

**Discussion by zeus82** [dotnet/aspnetcore#29959] [Q&A]
I was looking at the `Microsoft.AspNetCore.ApiAuthorization.IdentityServer.ConfigureClients` class, and for the `NativeApp` identity profile, you cannot specify redirect URLs.  Why is that?  There are several other Client settings that are not exposed, such as `AllowOfflineAccess`.

This limitation is making the code in my Startup.cs much more message than I feel it has to be.  It would be really nice to be able to fully configuration Identity Server clients in the appsettings.json.

**Comment by mrlife** [comment]
I think you will need to create an issue for this or post somewhere like Stack Overflow.

See: https://github.com/dotnet/aspnetcore/issues/29935
